Makefile

```diff
@@ -92,5 +92,14 @@ generate-api:
 vps:
 	VBoxManage list runningvms
 
+tt:
```

Comment:
Was hoping to keep it short, how about one of these?

`make reload`
`make change`
`make apply`